### PR TITLE
[AWIBOF-7108] Set telemetry isRunning to false when stopping telemetry

### PR DESCRIPTION
**Pull request by mjlee34**
Set telemetry isRunning to false when stopping telemetry.

Previously, the GETTELEMETRYPROPERTY command returns as if telemetry is running even though it's not.

The problem was because isRunning was not set to false in StopAllPoblisher().

This commit fixes the problem.

Signed-off-by: mjlee34 <jun20.lee@samsung.com>

### CODE REVIEW

File: src/telemetry/telemetry_client/telemetry_client.cpp [hunk 1]
```diff
@@ -143,6 +143,8 @@ TelemetryClient::StopAllPublisher(void)
         p.second->StopPublishing();
         POS_TRACE_DEBUG(EID(TELEMETRY_CLIENT_PUBLISH_STOP), "publisher:{}", p.first);
     }
+
+    isRunning = false;
```
Comment (hsung-yang) [inline src/telemetry/telemetry_client/telemetry_client.cpp:147]: global variable인가요?

**Review by hsung-yang — APPROVED** [review]
LGTM

**Review by aswinssir — APPROVED** [review]
LGTM. Thanks for the change

**Review by minjoon-ahn — APPROVED** [review]
LGTM

**Pull request merged**